core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1915,6 +1942,16 @@ private static List<ServerPort> resolveDistinctPorts(List<ServerPort> ports) {
         return Collections.unmodifiableList(distinctPorts);
     }
 
+    @SuppressWarnings("checkstyle:OverloadMethodsDeclarationOrder")
```

Comment:
Checkstyle complains about this because `dependencyInjector(DependencyInjector, ....)` and the getter `dependencyInjector` are not aligned. Let me rename the method `dependencyInjector()` to `dependencyInjectorOrReflective`. 😉

Reply:
Can we remove this now it's renamed?